<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TakePhotosClass</class>
 <widget class="QMainWindow" name="TakePhotosClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TakePhotos</string>
  </property>
  <property name="locale">
   <locale language="Armenian" country="Armenia"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="cameraLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>600</width>
      <height>481</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="locale">
     <locale language="Armenian" country="Armenia"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>200</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="transactorTab">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Transactor</string>
     </attribute>
     <widget class="QPushButton" name="cleanBtn">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Clean</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>180</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="readInfoBtn">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Read IDCard</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameInput">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>280</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="cardPhtotoTextLabel_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>180</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Photo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="cardPhotoLabel">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>35</y>
        <width>102</width>
        <height>126</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>180</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>ID Number</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="IDCardNumInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>160</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="agentTab">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="locale">
      <locale language="Armenian" country="Armenia"/>
     </property>
     <attribute name="title">
      <string>Agent</string>
     </attribute>
     <widget class="QLabel" name="agentPhotoLabel">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>35</y>
        <width>102</width>
        <height>126</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="cardPhtotoTextLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>180</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Photo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>180</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>ID Number</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>180</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="agentNameInput">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>280</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="readAgentBtn">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Read IDCard</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cleanAgentBtn">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Clean</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="agentNumInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>160</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="takePhotoBtn">
       <property name="text">
        <string>Take Photo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cleanAllBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Clean All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submitBtn">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>Reset IC card password</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>IC card relieves the loss</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Card - free settlement record</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>23</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuSwitch_Camera">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Switch Camera</string>
     </property>
    </widget>
    <addaction name="menuSwitch_Camera"/>
    <addaction name="actionFont_Settings"/>
    <addaction name="actionServer_Setting"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionHistory"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCamera_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera 1</string>
   </property>
  </action>
  <action name="actionCamera_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera 2</string>
   </property>
  </action>
  <action name="actionCamera_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera 3</string>
   </property>
  </action>
  <action name="actionCamera_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera 4</string>
   </property>
  </action>
  <action name="actionCamera_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera 5</string>
   </property>
  </action>
  <action name="actionText_Settings">
   <property name="text">
    <string>Text Settings</string>
   </property>
  </action>
  <action name="actionFont_Settings">
   <property name="text">
    <string>Font Settings</string>
   </property>
  </action>
  <action name="actionServer_Setting">
   <property name="text">
    <string>Server Setting</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>History</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBox_2</tabstop>
  <tabstop>takePhotoBtn</tabstop>
  <tabstop>IDCardNumInput</tabstop>
  <tabstop>nameInput</tabstop>
  <tabstop>readInfoBtn</tabstop>
  <tabstop>cleanBtn</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>agentNumInput</tabstop>
  <tabstop>agentNameInput</tabstop>
  <tabstop>readAgentBtn</tabstop>
  <tabstop>cleanAgentBtn</tabstop>
  <tabstop>cleanAllBtn</tabstop>
  <tabstop>submitBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
